SPECS/openldap/openldap.spec

```
@@ -15,9 +14,12 @@ Patch1:         openldap-2.4.44-consolidated-2.patch
 Patch2:         CVE-2015-3276.patch
 Patch3:         CVE-2021-27212.patch
 Patch4:         CVE-2022-29155.patch
+%global _default_patch_fuzz 2
+BuildRequires:  cyrus-sasl-devel >= 2.1
 BuildRequires:  e2fsprogs-devel
 BuildRequires:  groff
 BuildRequires:  openssl-devel >= 1.0.1
+Requires:       cyrus-sasl >= 2.1
```

It's possible that we can limit this to only `cyrus-sasl-lib`. This should limit the number of packages installed on images with OpenLDAP:
```suggestion
Requires:       cyrus-sasl-lib >= 2.1
```

The `Requires` in this case are mostly for the human reader - RPM automatically generates dependencies on any libraries. These can be viewed by running `rpm -qp --requires [path_to_openldap_rpm]`. This will dump a list of binaries and packages it depends on and I'm guessing for Cyrus SASL stuff that list will only contain dependencies provided by `cyrus-sasl-lib`.